README.md

```diff
@@ -197,3 +197,32 @@ var export = buffer.SelectUncommitted((id, verion, e) => Export(id, version, e))
 var updated = buffer.MarkAllAsCommitted();
 ```
 
+## Command Processor
+An approach often used when applying event sourcing is the [(command pattern)https://en.wikipedia.org/wiki/Command_pattern];
+every change on the domain(s), is described by a command, that is handled by 
```

Comment:
Every change on the domain(s) is described by a command, which is handled by a single command handler.